test/end-to-end/package.json

```diff
@@ -6,6 +6,7 @@
     "test": "jest --runInBand --forceExit"
   },
   "dependencies": {
+    "faker": "^4.1.0",
```

Comment:
Why do we need this?

Reply:
We need to generate a random blueprint description sentence to update for every test run. That'll avoid some cache issue which will impact string match with cached value but not new value.
faker is a good tool to generate all kinds of random things, like word, sentence, etc.

Reply:
In this case please update the commit log to describe why the commit adds a new dependency.

Reply:
OK, Thanks.